tests/integration/components/power-select/general-behaviour-test.js

```diff
@@ -891,23 +891,23 @@ module(
       assert.expect(6);
 
       let promise1 = new RSVP.Promise(function(resolve) {
-        run.later(resolve, numbers[3], 80);
+        run.later(resolve, numbers[3], 400);
       });
 
       let promise2 = new RSVP.Promise(function(resolve) {
-        run.later(resolve, numbers[4], 20);
+        run.later(resolve, numbers[4], 200);
       });
 
       this.numbers = numbers;
-      this.selected = promise1;
+      run(() => this.set('selected', promise1));
 
       await render(hbs`
         {{#power-select options=numbers selected=selected onchange=(action (mut foo)) as |option|}}
           {{option}}
         {{/power-select}}
       `);
 
-      this.set('selected', promise2);
+      run(() => this.set('selected', promise2));
```

Comment:
hmm, this.set should have already been run wrapped, I don’t think you should have to do it again 🤔

Reply:
And it didn't make a difference. I was just trying things.